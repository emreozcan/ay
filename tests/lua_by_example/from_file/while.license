https://luabyexample.techplexlabs.com/while/
Lua by Example: While
by Blake Bourque

This work is licensed under a Creative Commons Attribution 3.0 Unported License.