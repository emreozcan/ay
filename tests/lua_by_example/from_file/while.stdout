1
2
3
4
This loop will run forever.
